File: CS/Demos/10Xml/Person.cs
using System.Xml.Serialization;

namespace Demos.Xml;

public class Person
{
  [XmlAttribute("id")]
  public Guid Id { get; set; }
  public string? FirstName { get; set; }
  public string? LastName { get; set; }
  public int Number { get; set; }
  public string? Address { get; set; }
  public string? Place { get; set; }
  public string? State { get; set; }
  public string? Zip { get; set; }
}

File: VB/Demos/10Xml/Person.vb
Imports System.Xml.Serialization

Namespace Xml
  Public Class Person
    <XmlAttribute("id")>
    Public Property Id As Guid
    Public Property FirstName As String
    Public Property LastName As String
    Public Property Number As Integer
    Public Property Address As String
    Public Property Place As String
    Public Property State As String
    Public Property Zip As String
  End Class
End Namespace
